README.md

```diff
@@ -7,6 +7,7 @@ https://instagram.com/dsc_unhas
 ### DIMENSI STMIK DIPANEGARA
 ### XP STMIK DIPANEGARA
 ### CODER INSTITUTE UNHAS
+https://www.instagram.com/coderinstitute/
```

Comment:
`NIT`
```diff
- https://www.instagram.com/coderinstitute/
+ ### [Coder Institute](https://www.instagram.com/coderinstitute/)
```

Reply:
I don't think we can do this. See PR #7. Some community probably want to list multiple link. Or we just should limit to one community one URL?

Reply:
@rizkyarlin Okay, I don't think of some community with multiple link. 

Okay God job